content/preparingArtForBabylon/controllingColorSpace.md

```diff
@@ -0,0 +1,155 @@
+---
+title: How to Take Control of Color Space
+image: 
+description: Discover the considerations you should account for when planning your art pipeline for real time rendering.
+keywords: assets, workflow, pipeline, art, artwork, real-time rendering
+further-reading:
+video-overview:
+video-content:
+---
+
+### What is color space?

```

Comment:
it would be better to starts with h2 (`##`) so that the menu will be generated correctly. Unless you did that deliberately?

Reply:
Same to all following titles

Reply:
I am not a fan of the HR that is added for h2, but I can change it.

Reply:
the main importance is to have the menu rendered correctly (it stops at h3). we'll see how we can fix the h2 design.
Feel free to merge this if you find it enough as is